e3sm_to_cmip/cmor_handlers/utils.py

```diff
@@ -115,7 +115,7 @@ def _get_mpas_handlers(cmip_vars: List[str]):
 
         derived_handlers.append(var_handler[0])
 
-    if len(missing_handlers) > 0:
+    if len(missing_handlers) == len(cimp_vars):
         raise KeyError(
             f"No variable handlers are defined for {missing_handlers}. Make sure at "
             "least one variable handler is defined for each of these variables in "
```

Comment:
```suggestion
    if len(missing_handlers) == len(cimp_vars):
        logger.warning(
            f"No variable handlers are defined for {missing_handlers}. Make sure at "
            "least one variable handler is defined for each of these variables in "
```
Same comment as above.